src/styles/styles.js

```diff
@@ -783,6 +783,15 @@ const styles = {
         justifyContent: 'center',
     },
 
+    chatItemTopPadding: {
+        backgroundColor: 'rgba(0,0,0,0)',
+        textAlignVertical: 'top',
+        paddingTop: 0,
+        top: 10,
```

Comment:
I'm curious if there is any explanation as to why this solves the issue? Why can we not use padding? It seems like we should be able to.

Reply:
The bug on Android is linked to the line below. It's the `padding:10` at the moment that has no effect even when removed. 
https://github.com/Expensify/Expensify.cash/blob/9147d3f469a24aeeaf786e174fcbf6bc33a2c960/src/styles/styles.js#L610

Reply:
This doesn't seem correct. Please see my most recent comment about this likely being related to the `height: 'auto'` property. If you remove it there is no extra padding - and it seems the padding values we have now are not influencing this either.

Reply:
I get it. Thanks